recipe/build.sh

```diff
@@ -17,7 +16,22 @@ export FC=$(basename "$FC")
     --with-includes=$PREFIX/include \
     --with-openssl \
     --with-gssapi  \
+    --with-icu \
+    --with-ldap \
+    --with-libxml \
+    --with-libxslt \
+    --with-lz4 \
+    --with-zstd \
+    --with-uuid=e2fs \
+    --with-system-tzdata=$PREFIX/share/zoneinfo \
     PG_SYSROOT="undefined"
 
 make -j $CPU_COUNT
-# make check # Failing with 'initdb: cannot be run as root'.
```

Comment:
I'm curious, why don't we build with meson on unix?

Reply:
No particular reason. This was my first experience with it as a build system and only found it necessary to replace the previous perl-based builds on Windows. Since the previous autotools configuration still worked with v17 I left it in place.